README.md

```diff
@@ -62,6 +62,18 @@ You can now see the developer center at <http://localhost:4000>.
 
 Note: the header and footer may not look right. This is inconsequential as they are stripped from the page before embedding into the webserver.
 
+### Running the Node server locally
+
+ 1. First, build the `./sites` folder to serve content out of:
+
+    ```
+    mkdir /sites && \
```

Comment:
Is there any chance we could make this a script instead of requiring the copy-pasting?

Reply:
I like that. [Updated.](https://github.com/algorithmiaio/dev-center/pull/210/commits/f42c992d1cf779b7abc6f8444df2f13a6fe3290c)